README.md

```diff
@@ -270,13 +285,45 @@ pip install -r requirements-example.txt
 python examples/ckan-upload-csv.py
 ```
 
-## Tutorials
+## 
 
 ### Using Google Cloud Composer
 
-* Sign up for an account.
-* Create a bucket for storing results and log data (automatic IIRC).
-* Upload this library code...
-* Set up the DAG...
-* Configure with this file online: https://raw.githubusercontent.com/datopian/aircan/examples/...
-* Run it...
+1. Sign up for an account at https://cloud.google.com/composer. Create or select an existing project at Google Cloud Platform. For this example, we use one called `aircan-test-project`.
+2. Create an environment at Google Cloud Composer, either by command line or by UI. Make sure you select **Python 3** when creating the project. Here, we create an environment named `aircan-airflow`.
+![Google Cloud Composer environment configuration](docs/resources/images/composer.png)
```

Comment:
404 -not found -  [Google Cloud Composer environment configuration](https://github.com/datopian/aircan/blob/master/docs/resources/images/composer.png)